examples/platforms/nrf528xx/src/radio.c

```diff
@@ -1362,3 +1363,29 @@ int8_t nrf5GetChannelMaxTransmitPower(uint8_t aChannel)
 
     return power;
 }
+
+otError otPlatRadioSetRegion(otInstance *aInstance, uint16_t aRegionCode)
+{
+    OT_UNUSED_VARIABLE(aInstance);
+
+    sRegionCode = aRegionCode;
+    nrf5HandleRegionChanged(aRegionCode);
+    return OT_ERROR_NONE;
+}
+
+otError otPlatRadioGetRegion(otInstance *aInstance, uint16_t *aRegionCode)
```

Comment:
```suggestion
uint16_t otPlatRadioGetRegion(otInstance *aInstance, uint16_t *aRegionCode)
```

Reply:
On posix platforms there will be spinel errors so I tend to return the error code rather than only printing it.